mzLib/MassSpectrometry/Deconvolution/AverageResidue/OxyriboAveragtide.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using Chemistry;
+
+namespace MassSpectrometry;
+
+/// <summary>
+/// Represents the average RNA nucleotide and is used for most abundant isotopic peak to monoisotopic peak difference
+/// </summary>
+/// <remarks>All instance methods return a reference to its static precalculated values</remarks>
+public sealed class OxyriboAveragtide : AverageResidue
+{
+    public static readonly double[][] AllMasses = new double[NumAveraginesToGenerate][];
+    public static readonly double[][] AllIntensities = new double[NumAveraginesToGenerate][];
+    public static readonly double[] MostIntenseMasses = new double[NumAveraginesToGenerate];
+    public static readonly double[] DiffToMonoisotopic = new double[NumAveraginesToGenerate];
+    public override int GetMostIntenseMassIndex(double testMass) => MostIntenseMasses.GetClosestIndex(testMass);
+
+    public override double[] GetAllTheoreticalMasses(int index) => AllMasses[index];
+
+    public override double[] GetAllTheoreticalIntensities(int index) => AllIntensities[index];
+
+    public override double GetDiffToMonoisotopic(int index) => DiffToMonoisotopic[index];
+    static OxyriboAveragtide()
+    {
+        // Magic numbers determined by counting atoms in the main 4 canonical RNA bases. 
+        double averageC = 9.5;
```

Comment:
these numbers are quite "low-res" compared to the same values in the protein averagine

Reply:
This is an extremely oversimplistic approach and should be further refined.